docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -84,14 +84,16 @@ SQL support
 -----------
 
 The connector provides **read access and limited write access** to data and
-metadata in Iceberg. The following section lists all supported SQL statements
-and provides further details.
+metadata in Iceberg. The following section lists supported SQL statements and
+provides further details.
 
-  .. important::
+.. important::
 
-      No other SQL statements are supported.
+    Only SQL statements in the following lists and the :ref:`globally available
+    statements <sql-globally-available>` are supported. No other SQL statements
+    are supported.
```

Comment:
This connector supports the following SQL statements in addition to the :ref:globally available statements <sql-globally-available>. No other SQL statements are supported.